### DISCUSSION TITLE
Issues with "dunstrc" retrieving an "svg" icon from my font awesome download

### DISCUSSION THREAD

**Discussion by wild-d0g** [dunst-project/dunst#1368] [Help]
I've been trying to test some custom notifications, along with icons I've downloaded from "Fontawsome" however whenever I attempt to test my notification with the icon I downloaded the icon will not appear--this is even after making the appropriate adjustments to the " /path/to/icon" within my dunstrc file. Has anyone encountered a similar issue before? I've seen a similar discussion, but there is no information on how to solve it

**Comment by bynect** [comment]
Does sending the icon explicitly work? Like `dunstcl -i path hello`?
Also could you send the dunstrc

**Reply by wild-d0g** [reply]
:/ unfortunately that doesn't seem to work for me--I ran "dunstcl -i  /home/joe/.local/share/icons/fontawesome/keyboard-regular.svg hello" but It didn't execute for me

**Reply by bynect** [reply]
> :/ unfortunately that doesn't seem to work for me--I ran "dunstcl -i  /home/joe/.local/share/icons/fontawesome/keyboard-regular.svg hello" but It didn't execute for me

Sorry I made a typo 😬. I meant `dunstify` instead of dunstctl. Or you could try with notify-send

**Reply by wild-d0g** [reply]
Unfortunately only the text for the--notification is showing up

**Reply by bynect** [reply]
> Unfortunately only the text for the--notification is showing up

I see. Have you installed librsvg? It is a runtime dependency for loading svg. In any case could you start dunst with debug logging and send me the logs
`dunst -verbosity debug`
